zigup.zig

```diff
@@ -136,6 +136,8 @@ fn help() void {
         \\  zigup VERSION                 download and set VERSION compiler as default
         \\  zigup fetch VERSION           download VERSION compiler
         \\  zigup default [VERSION]       get or set the default compiler
+        \\  zigup clean   [VERSION]       clean the installed compilers that are not marked as keep or master. if a version is specified, it will clean that version
```

Comment:
This could be worded a bit better
```
        \\  zigup clean   [VERSION]       deletes the given compiler version, otherwise, cleans all compilers 
        \\                                that aren't the default, master, or marked to keep.
```